app/static/js/userManagement.js

```diff
@@ -82,7 +82,13 @@ function addNewTerm(){
     url: "/admin/addNewTerm",
     type: "POST",
     success: function(s){
-      location.reload()
+      var origin = window.location.href;
+
+      if (origin.includes("?")){
+        origin = origin.slice(0, origin.indexOf("?"));
+      }
+
+      location.replace(origin + "?accordion=term")
```

Comment:
We should make this a function so we can just call `reloadWithAccordion("term")`